guides/v2.2/ui_comp_guide/components/ui-fieldset.md

```diff
@@ -20,8 +20,46 @@ The Fieldset component implements a container for visually-grouped form elements
 | Title         | Type    | Default | Description                                                                       |
 | ------------- | ------- | ------- | --------------------------------------------------------------------------------- |
 | `collapsible` | Boolean | `false` | Enables/disables the collapsible functionality.                                   |
+| `label`       | String  |         | Caption for the fieldset.                                                         |
 | `opened`      | Boolean | `false` | Initial collapsible state. Applied when the collapsible functionality is enabled. |
 
+## Examples
+
```

Comment:
Please, add another subheader that explains the following example. Refer the [following](https://github.com/magento/devdocs/pull/6039/files#diff-4204d17776bb584fd954af5edac31c25R28) PR as an example.

Thank you

Reply:
Hi @rogyar, the subheader has been added.
Thank you.